lib/index.js

```diff
@@ -35,5 +43,63 @@ Parsely.prototype.initialize = function() {
 };
 
 Parsely.prototype.loaded = function() {
-  return !!window.PARSELY;
```

Comment:
since we stub this now (so we can add `autotrack: false` if necessary), we need to rely on a field that gets added by their script executing